src/main/java/com/codeborne/selenide/Selenide.java

```diff
@@ -101,6 +104,35 @@ public static void open(String relativeOrAbsoluteUrl, String domain, String logi
     mockModalDialogs();
   }
 
+  /**
+   *
+   * The main starting point in your tests.
+   * <p>
+   * Open browser and pass authentication using build-in proxy.
+   * <p>
+   * A common authenticationType is "Basic". See Web HTTP reference for other types.
+   * <p>
+   * {@code Configuration.fileDownload == Configuration.FileDownloadMode.PROXY;} - is required configuration.
+   *
+   * @param relativeOrAbsoluteUrl
+   * @param authenticationType
+   * @param login
+   * @param password
+   *
+   * @see <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Proxy-Authorization">Web HTTP reference</a>
+   * @see AuthenticationType
+   */
+  public static void open(final String relativeOrAbsoluteUrl, final AuthenticationType authenticationType,
+                          final String login, final String password) {
+    getAndCheckWebDriver();
+    final BrowserMobProxy proxy = getSelenideProxy().getProxy();
+    final Credentials credentials = new Credentials(login, password);
+    final String authorization = String.format("%s %s", authenticationType.getValue(), credentials.encode());
```

Comment:
I personally do not like `final` near every local variable and method parameter. It's a noise.